docs/developers/developing_iree/benchmark_suites.md

```diff
@@ -187,17 +187,26 @@ more usages):
 
 ```sh
 # The GCS directory has the same structure as your local ${IREE_BUILD_DIR}/e2e_test_artifacts.
-gcloud storage ls gs://iree-github-actions-postsubmit-artifacts/.../.../e2e-test-artifacts
+gcloud storage ls gs://iree-github-actions-...-artifacts/.../.../e2e-test-artifacts
```

Comment:
Are these `...`s literally those characters, or are those standing in for some other text? Can you make these easier to copy-paste if so? (list what `...` can be in each case, or demonstrate with a sample what should go there)

Reply:
Done. Define a variable first and use it in the following commands.